woocommerce/wc-admin
# Extend report submenu items

Allow extensions to add a submenu item to the sidebar 

![Screen Shot 2019-04-10 at 4 13 31 PM](https://user-images.githubusercontent.com/1922453/55851506-fbe37080-5bac-11e9-8f6a-cd742df0a2f6.png)

This PR is also an example of how https://github.com/woocommerce/woocommerce-admin/pull/2018 could be useful in developing and testing extensibility because the code and example reside in the same pull request.

### Detailed test instructions:

1. `npm run example -- --ext=add-report`.
1. Activate the WooCommerce Admin Add Report Example plugin.
2. Open wc-admin and see the "Example" report menu item appear in the sidebar.
3. Make sure the link works.
